### ISSUE TITLE
Doesnt manage nested dirs correctly

### ISSUE THREAD

**Issue by AlexandreBonaventure** [alwaysonlinetxm/rollup-plugin-fill-html#6]
Hello, 
I've noticed html isn't copied in the right place when using nested directories in output path.
Here's a fork showing the bug (you'll have to run your example), you'll see that 

index.html -> /bug/dist/index.html
**What is expected ?**
That index.html is at the same location than the bundle so :
index.html -> /bug/dist/simple/index.html

**Comment by alwaysonlinetxm** [comment]
@AlexandreBonaventure Yes, you are right. But it's the default behavior of `rollup-plugin-fill-html`. That's because I think the result html should just be in the root of a website, and the other resource files can be in nested dirs. Think about this, you generate a html like ` /bug/dist/simple/index.html`, and some resource files, maybe ` /bug/dist/bundle.js`. So what will we see in the html? 
```JavaScript
<script type="text/javascript" src="../bundle.js"></script>
```
I think it's not a good way to use '../' in resource file reference.
